rs/tests/message_routing/xnet/BUILD.bazel

```diff
@@ -5,6 +5,9 @@ package(default_visibility = ["//rs:system-tests-pkg"])
 
 system_test(
     name = "xnet_slo_120_subnets_staging_test",
+    env = {
+        "XNET_TEST_CANISTER_WASM_PATH": "$(rootpath //rs/rust_canisters/xnet_test:xnet-test-canister)",
```

Comment:
Shouldn't there be a place where we actually use this env variable?

Reply:
We use it [here](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/dfinity/ic%24+XNET_TEST_CANISTER_WASM_PATH+file:.*.rs&patternType=keyword&sm=0).